components/card.css

```diff
@@ -24,11 +29,81 @@
 .card-title {
 	margin: 1rem 0;
 }
+/* CARD PRICE FOR ECOM */
 .card-price-striked {
 	text-decoration: line-through;
 	margin: 1rem;
-	/* color: #ccc; */
+	font-size: 1.3rem;
+	color: var(--color-grey-dark);
 }
 .card-price {
-	font-size: 1.5rem;
+	font-size: 1.6rem;
+	color: var(--color-primary);
+}
+
+/* BLOG CARD */
+.card-container {
+	display: flex;
+	flex-direction: column;
+	margin: 1rem;
+	padding: 1rem;
+	width: 36rem;
+	position: relative;
+	background-color: var(--color-white);
+	-webkit-box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+	-moz-box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+	box-shadow: -1px 1px 15px 0px rgba(176, 176, 176, 1);
+}
+.img-box,
+img {
+	width: 100%;
+}
+
+.card-heading-primary {
+	margin: 0;
+}
+.card-heading-sub {
+	color: var(--color-grey-dark);
+	margin: 0;
+}
+.card-footer {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+.overlay {
+	position: absolute;
+	color: var(--color-white);
+	top: 50%;
+	left: 6%;
+}
+.heading-top {
+	order: -1;
+}
+.flex {
+	display: flex;
+}
+.small-img {
+	/* height: 110px; */
+	width: 25%;
+}
+/* SMALL CARD MARGIN */
+.card-header-small {
+	margin-left: 1rem;
+}
+/* CARD BADGE */
+.card-badge {
+	position: absolute;
+	background-color: var(--color-primary);
+	color: var(--color-white);
+	padding: 7px 1.3rem;
+}
+/* CARD DISMISS BTN */
+.card-dismiss {
+	position: absolute;
+	color: var(--color-white);
+	padding: 7px 1.3rem;
+	padding: 7px 1.3rem;
+	right: 0rem;
```

Comment:
not necessary to mention unit when the value is 0

Reply:
Removed it